### PR TITLE
34 project profile

### PR DESCRIPTION
**Pull request by niver-copa-jalafund**
Hi Guys, I was end this part from the project, if you have any cuestions, let me know

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+  return (
+    <Router>
+      <div className="App">
+        <Switch>
+          <Route path="/project/:id">
+            <Project />
+          </Route>
+          <Route path="/">
+             <ProjectForm />
+          </Route>
```
Comment (jorge-lema-jalafund) [inline src/components/App/App.jsx]: This route-path is wrong and also it's not part of your job.

File: src/components/ProjectForm/ProjectDetail.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import { useParams } from "react-router-dom";
+import {
+  Button,
+  Container,
+  Divider,
+  Grid,
+  Typography,
+} from "@material-ui/core";
+import React, { useEffect, useState } from "react";
+const Project = () => {
```
Comment (jorge-lema-jalafund) [inline src/components/ProjectForm/ProjectDetail.jsx:12]: Please leave an empty line between imports and the rest of the code

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

File: src/components/ProjectForm/ProjectDetail.jsx (hunk 2 above)
Comment (niver-copa-jalafund) [reply src/components/ProjectForm/ProjectDetail.jsx:12]: This part is Solved

File: src/components/App/App.jsx (hunk 1 above)
Comment (niver-copa-jalafund) [reply src/components/App/App.jsx]: Solved

**Review by lilian-delgado-jalafund — DISMISSED** [review]

**Comment by DaleneyRuben** [comment]
@niver-copa-jalafund Please update your branch with develop

**Pull request closed without merging**